logging/access.go

```diff
@@ -82,14 +85,16 @@ func omitWhitespace(h string) string {
 
 func (f *accessLogFormatter) Format(e *logrus.Entry) ([]byte, error) {
 	keys := []string{
-		"host", "timestamp", "method", "uri", "proto",
+		"host", "auth-user", "timestamp", "method", "uri", "proto",
 		"status", "response-size", "referer", "user-agent",
 		"duration", "requested-host", "flow-id", "audit"}
 
 	values := make([]interface{}, len(keys))
 	for i, key := range keys {
 		if s, ok := e.Data[key].(string); ok {
 			values[i] = omitWhitespace(s)
+		} else if e.Data[key] == nil {
+			values[i] = "-"
 		} else {
 			values[i] = e.Data[key]
 		}
```

Comment:
I think this should not be changed at all - `omitWhitespace` changes empty value for `-`